tests/tests_suite_test.go

```diff
@@ -25,6 +25,11 @@ type Cmd struct {
 	CommandLineString string
 }
 
+type App struct {
```

Comment:
What do we think of using an extensible `App` struct for `testApp` instances, as opposed to the currently used/not quite as powerful `appName` string?